includes/admin/post-types/meta-boxes/class.llms.meta.box.award.engagement.submit.php

```diff
@@ -91,6 +231,33 @@ public function redirect_on_deletion( $location ) {
 
 	}
 
+	/**
+	 * Retrieve the current student id.
+	 *
+	 * @since [version]
+	 *
+	 * @param null|bool $creating Whether or not we're awarding an engagement.
+	 *                            If not provided, it'll be dynamically retrieved if the current screen's action is 'add'. See WordPress' `get_current_screen()`.
+	 * @return int
+	 */
+	private function current_student_id( $creating = null ) {
```

Comment:
Tests can (and should) be written for this method.

Reply:
done